scipy/cluster/tests/test_hierarchy.py

```diff
@@ -443,21 +443,21 @@ def test_is_valid_linkage_various_size(self, nrow, ncol, valid, xp):
         Z = xp.asarray([[0, 1, 3.0, 2, 5],
                         [3, 2, 4.0, 3, 3]], dtype=xp.float64)
         Z = Z[:nrow, :ncol]
-        xp_assert_equal(is_valid_linkage(Z), valid, check_namespace=False)
+        assert is_valid_linkage(Z) == valid
```

Comment:
I am not sure why the tests were using `xp_assert_equal` here. I will find it out through the reviews or the failing CI 🙂